README.md

```diff
@@ -1,13 +1,186 @@
+
+### Code Along I: INSERT INTO
+
+In your terminal, in the sqlite prompt, type the following: 
+
+```sql
+INSERT INTO cats (name, age, breed) VALUES ('Maru', 3, 'Scottish Fold');
+```
+
+We use the `INSERT INTO` command, followed by the name of the table to which we want to add data. Then, in parentheses, we put the column names that we will be filling with data. This is followed by the `VALUES` keyword, which is accompanied by a parentheses enclosed list of the values that correspond to each column name. 
+
+**Important:** Note that we *didn't specify* the "id" column name or value. Since we created the `cats` table with an "id" column whose type is `INTEGER PRIMARY KEY`, we don't have to specify the id column values when we insert data. Primary Key columns are auto-incrementing. As long as you have defined an id column with a data type of `INTEGER PRIMARY KEY`, a newly inserted row's id column will be automatically given the correct value. 
+
+Let's add a few more cats to our table. Create a file, `01_insert_cats_into_cats_table.sql`. Use several `INSERT INTO` statements to insert the following cats into the table:
+
+|id |name| age| breed|
+|---|----|----|------|
+|  1 | "Lil' Bub" | 5 | American Shorthair|
+|  2  | "Hannah" | 1 | Tabby|
+
+Each `INSERT INTO` statement gets it's own line in the `.sql` file in your text editor. Each line needs to end with a `;`. 
+
+Now, we'll learn how to `SELECT` data from a table, which will help us to confirm that we inserted the above data correctly. 
+
+## Selecting Data
+
+Now that we've inserted some data into our `cats` table, we likely want to read that data. This is where the `SELECT` statement comes it. We use it to retrieve database data, or rows. 
+
+### Code Along II: SELECT FROM
+
+A basic `SELECT` statement works like this: 
+
+```sql
+SELECT [names of columns we are going to select] FROM [table we are selecting from];
+```
+
+We specifiy the names of the columns we want to SELECT and then tell SQL the table we want to select them FROM. It's fairly straight forward.
+
+For this particular select, we are also going to be making use of a special character. We want to select all the rows in our table, and we want to return the data stored in any and all columns in those rows. To do this, we could pass the name of each column explicitly:
+
+```sql
+SELECT name, age, breed FROM cats;
+```
+
+Which should give us back:
+
+```bash
+1 | Maru     | 3 | Scottish Fold
+2 | Hannah   | 1 | Tabby
+3 | Lil' Bub | 5 | American Shorthair
+```
+
+But, since we aren't trying to explicitly list the columns we want to select in this case, and are really trying to say "Give me all the data from all the columns for all of the wizards" we can make use of a special selector, known commonly as the "wildcard" selector. You can think of this selector as simply meaning "all" or anything. It is represented by an asterix like `*`. So, to `SELECT` all the data from all of the cats	 we would say:
```

Comment:
I might reword this line as "A faster way to do this is using a special selector, know commonly as the 'wildcard' selector. You can think of this..."